BUILD.gn

```diff
@@ -53,7 +53,8 @@ if (!build_with_chromium && !build_with_woogeen) {
         "modules/audio_processing:audio_processing_tests",
         "modules/remote_bitrate_estimator:bwe_simulations_tests",
         "modules/rtp_rtcp:test_packet_masks_metrics",
-        "modules/video_capture:video_capture_internal_impl",
+        #"modules/video_capture:video_capture_internal_impl",
```

Comment:
Please delete the code you want to disable. Git keeps track of code history.

Reply:
done